CODE_OF_CONDUCT.md

```diff
@@ -2,50 +2,45 @@
 
 ## Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
-size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+As members, contributors, and leaders, we pledge to ensure that everyone can participate 
+in our community without harassment regardless of age, body size, visible or invisible disability,
+ethnicity, sex characteristics, gender identity, or level of experience.
+
 
 We pledge to act and interact in ways that contribute to an open, welcoming,
-diverse, inclusive, and healthy community.
+diverse, inclusive, diverse and healthy community.
 
 ## Our Standards
 
-Examples of behavior that contributes to a positive environment for our
-community include:
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
+Examples of behaviour that contributes to a positive environment for our community include:
+
+* Demonstrating empathy and kindness toward other people.
+* Being respectful of opposing viewpoints, perspectives, and experiences.
+* Giving and receiving constructive feedback with grace. 
 * Accepting responsibility and apologizing to those affected by our mistakes,
-  and learning from the experience
-* Focusing on what is best not just for us as individuals, but for the
-  overall community
+  and learning from the experience.
+* Focusing on what is best not just for us as individuals but for the
+  overall community.
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behaviour include:
```

Comment:
There is not an agreement in the core maintainers of using british english. Most of the codebase and documentation is written in american english so this change it feels opinionated.